components/x-live-blog-post/readme.md

```diff
@@ -52,3 +52,6 @@ Feature             | Type   | Notes
 `publishedTimestamp`| String | Deprecated - ISO timestamp of publish date
 `articleUrl`        | String | Url of the main article that includes this post
 `showShareButtons`  | Bool   | default: `false` - Shows social media share buttons when `true`
+`topRef`  			| String   | Shows the back to top link at the bottom of posts and navigates to the hashed section of the page. No defaults, example `#top`
+`backToTop`  		| Function   | Shows the back to top link at the bottom of posts and manages navigating to `selected top` with a javascript function. No defaults. Only pass in `topRef` or `backToTop` props to the component. Please call event.preventDefault() at the top level of this function.
```

Comment:
**Question:** Are there reasons why we have decided to make this a function (maybe something in the app?) rather than just allowing the browser to handle the navigating to the top hash location?

Reply:
Sorry - I just remembered you mentioned this in the PR description (I forgot whilst reading code)

Reply:
**Question:** Is there a requirement for 2 type of behaviour at this stage?

**Feedback:** I am not sure the options (`topRef ` & `backToTop `) are obvious that they related bit provide different functionality. I think it might be nicer to make this clearer if we really need more sets of functionality

Reply:
In terms of the browser controls scrolling to the top, I assumed that in itself may be the best user experience. Since the function is going to be defined by the consumer, I figured it wasn't much work to add a callback function as well. Also a thing Olga may appreciate too.

For the names, it's difficult to tell without reading the docs. So I'll try `backToTopRef` and `backToTopFunction`. That should give more context.

Reply:
That sounds reasonable - I just always question building features that may never get used. If they are not used its just extra functionality to have. I agree it might be nicer to let apps manage the scroll and I think in the app (might be wrong) that might have some other benefits about keeping that position which is why we are building this.

This is just a suggestion so feel free to ignore it. What if we just had 1 option called `backToTop`but could accept either a function or a string? I think this might make the docs clearer that theres a single back to top option but if you want to manage scrolling yourself you pass it a function instead of a string.

From reading the docs as they are right now I at first thought to get back to top functionality I needed to pass in both `topRef` & `backToTop`

Reply:
Agreed. It's a robust solution. I'll change the implementation. Thanks this helps a lot.